Python/Dynamic-Programming/Travelling Salesman Problem/tsp.py

```diff
@@ -0,0 +1,29 @@
+from sys import maxsize
+from itertools import permutations
+V = 4
+

```

Comment:
write some comment lines explaining the approach

Reply:
ok